code/reports/rmdchunks/model-ranking-plots.Rmd

```diff
@@ -0,0 +1,42 @@
+```{r, results = 'asis'}
+library(forcats)
+
+lookup_variables <- names(target_variables)
+names(lookup_variables) <- target_variables
+skill_table <- table %>%
+  mutate(variable = lookup_variables[target_variable])
```

Comment:
```suggestion
skill_table <- table %>%
  mutate(variable = fct_recode(target_variable, !!!target_variables))
```

Reply:
nice - we'll need to add a `forcats` dependency, right?